docker/maistra-builder_2.4.Dockerfile

```diff
@@ -36,7 +36,7 @@ ENV PROTOC_GEN_SWAGGER_VERSION=v1.8.6
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV BENCHSTAT_VERSION=9c9101da8316
 ENV GH_VERSION=2.17.0
-ENV K8S_TEST_INFRA_VERSION=b3a9f2479e
+ENV K8S_TEST_INFRA_VERSION=b3a9f2479edd51c1da898c2073bab8512de275ec
```

Comment:
This affects the script we use to update prow: https://github.com/maistra/test-infra/blob/main/tools/bump_prow.sh
It gets this [short] value directly from upstream istio.

Reply:
Upstream istio uses long version as far as I can tell  https://github.com/istio/tools/blob/a8ec01a2ce2e67af0e3e81caba6fb3da102737a7/docker/build-tools/Dockerfile#L57

Reply:
I know. Look at the script I linked. It automatically replaces this value in Dockerfile with the one we grab from thie line https://github.com/istio/test-infra/blob/master/prow/cluster/deck_deployment.yaml#L27

Reply:
Ah ok, gotchya.

I will bundle all of it here, in one PR once fixed.

Reply:
217f498